config/routes.rb

```diff
@@ -5,6 +5,8 @@
 get 'welcome/sleepandfood', to: 'welcome#sleepandfood'
 get 'welcome/relationship',to: 'welcome#relationship'
 get 'welcome/parenting', to: 'welcome#parenting'
+resources :comments
+get 'welcome/comments', to: 'comments#new'
```

Comment:
arey localhost:3000/comments/new 
aisa path dene mai mera output aa ja rha h...par mjhe phle welcome page kholna h..phir health mai click krna h to nya page khulega..us page mai comment likhne hain